src/Microsoft.Azure.SignalR.Common/Auth/ApplicationOptions.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+using System.Threading.Tasks;
+using Microsoft.Azure.SignalR.Common.Auth;
+
+namespace Microsoft.Azure.SignalR
+{
+    public class ApplicationOptions : AuthOptions, ITokenBasedAuthOptions
```

Comment:
`Application` has special meaning in SDK, it is the application. `AadAuthOptions`?